connectors/sap-ariba/README.md

```diff
@@ -1,5 +1,170 @@
 ---
-fidelity: IDEA
+fidelity: GUIDE
 name: SAP Ariba
-
+difficulty_level: INTERMEDIATE
+time_in_minutes: 60
+accreditations:
+- DEFAULT
 ---
+
+# Introduction
+
+SAP Ariba is a comprehensive cloud-based solution for procurement and supply chain management. This guide covers the access needs that are required to establish a stable connection to your Ariba Instance, authenticate with SAP Ariba’s APIs and test it successfully in Creator Studio.
```

Comment:
Add a comma after "SAP Ariba's APIs", so it should be This guide covers the access needs that are required to establish a stable connection to your Ariba Instance, authenticate with SAP Ariba’s APIs**,** and test it successfully in Creator Studio.